zc_events/config.py

```diff
@@ -54,6 +50,9 @@ def __getattr__(self, name):
                         )
                     )
                     raise e
+            elif django_settings:
```

Comment:
Although unlikely, what would happen if `import_path` and `django_settings` both evaluated to `False`?

Reply:
This was fixed.